apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationPreviewList.tsx

```diff
@@ -0,0 +1,578 @@
+  ReviewApplicationPreviewListFragment,
+} from "@gc-digital-talent/graphql";
+import {
+  commonMessages,
+  getLocale,
+  getLocalizedName,
+} from "@gc-digital-talent/i18n";
+import {
+  Accordion,
+  Button,
+  Dialog,
+  Link,
+  PreviewList,
+  PreviewMetaData,
+  Separator,
+  Well,
+} from "@gc-digital-talent/ui";
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+
+import FieldDisplay from "~/components/ToggleForm/FieldDisplay";
+import talentRequestMessages from "~/messages/talentRequestMessages";
+import processMessages from "~/messages/processMessages";
+import { getClassificationName } from "~/utils/poolUtils";
+import {
+  ApplicationStatus,
+  getApplicationStatusChip,
+  getSalaryRange,
+} from "~/utils/poolCandidate";
+import { sortPoolSkillsBySkillCategory } from "~/utils/skillUtils";
+import useRoutes from "~/hooks/useRoutes";
+
+import { ApplicationDate } from "./MetadataDate";
+import StatusSummary from "./StatusSummary";
+
+export const ReviewApplicationPreviewList_Fragment = graphql(/* GraphQL */ `
+  fragment ReviewApplicationPreviewList on PoolCandidate {
+    id
+    expiryDate
+    finalDecisionAt
+    submittedAt
+    removedAt
+    status {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    finalDecision {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    assessmentStatus {
+      assessmentStepStatuses {
+        step
+      }
+      overallAssessmentStatus
+      currentStep
+    }
+    pool {
+      id
+      name {
+        en
+        fr
+      }
+      classification {
+        group
+        level
+        minSalary
+        maxSalary
+      }
+      workStream {
+        name {
+          en
+          fr
+        }
+      }
+      language {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      department {
+        name {
+          en
+          fr
+        }
+      }
+      closingDate
+      opportunityLength {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      isRemote
+      location {
+        en
+        fr
+      }
+      securityClearance {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      areaOfSelection {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      processNumber
+      poolSkills {
+        id
+        type {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        skill {
+          id
+          key
+          name {
+            en
+            fr
+          }
+          category {
+            value
+            label {
+              en
+              fr
+            }
+          }
+        }
+      }
+      screeningQuestions {
+        id
+      }
+    }
+  }
+`);
+
+interface ReviewApplicationDialogProps {
+  applicationQuery: ReviewApplicationPreviewListFragment;
```

Comment:
I think the type is wrong here. Typically it looks like:

```
  applicationQuery: FragmentType<ReviewApplicationPreviewList_Fragment>;
````

So we get the benefits of fragment masking for type checking.

Reply:
Changed in [b7e4eb3](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/b7e4eb30269e46bbc3948d9cbf267c7c15a45111)

Reply:
Fixed in [988d3bb](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/988d3bb22efa2d2da6544b633783b3c70a3f7ab5)